dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -604,7 +604,7 @@ export -f diff-apply
 mkdir temp-installer || true
 pushd temp-installer
 # this will split the big yaml produced by the installer, so we can diff individual parts of it and run them in parallel
-yq4 -s '.kind + "_" + .metadata.name' "../${INSTALLER_RENDER_PATH}"
+yq4 -s '.kind + "_" + .metadata.namespace // "" + "_" + .metadata.name' "../${INSTALLER_RENDER_PATH}"
```

Comment:
`// ""` defaults to an empty string if the namespace field isn't set (e.g. for namespaces 🙃), otherwise those resources would get skipped